lib/partial/transmuxer.js

```diff
@@ -27,38 +27,34 @@ var tsPipeline = function(options) {
       packet: new m2ts.TransportPacketStream(),
       parse: new m2ts.TransportParseStream(),
       elementary: new m2ts.ElementaryStream(),
-      videoRollover: new m2ts.TimestampRolloverStream('video'),
```

Comment:
this ends up matching up our non-partial transmuxer, right?

Reply:
Yeah, this was the rollover fix pat did awhile ago, but it never made it to the partial transmuxer. Without this the tests, and certainly some streams, were failing because the dts time for audio was much higher (never rolled over) then the video